CHANGELOG

```diff
@@ -0,0 +1,10 @@
+Arduino_MKRENV ?.?.? - ????.??.??
+
+Arduino_MKRENV 1.1.0 - 2019.05.31
```

Comment:
Just change to today's date, and we should be good :)

Reply:
done